root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2734,6 +2734,12 @@ limited_link_type_combinations: [
     expected_relationship_type: 'socialnetwork',
             expected_clean_url: 'https://www.linkedin.com/in/legselectric',
   },
+  {
+                     input_url: 'https://ca.linkedin.com/in/didier-charette-0630b1b6?original_referer=https%3A%2F%2Fduckduckgo.com%2F',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'socialnetwork',
+            expected_clean_url: 'https://www.linkedin.com/in/didier-charette-0630b1b6',
+  },
```

Comment:
* The previous test with `legselectric` might be no longer needed as it’s quite redundant with this one.
* The earlier test with `trevor-muzzy` would be worth an `expected_relationship_type`.

Reply:
It already has `expected_relationship_type` so I'm going to assume you meant `expected_clean_url`?

Reply:
Yes, thank you!